tests/test_views.py

```diff
@@ -971,8 +973,12 @@ def _login(self):
     @mock.patch('enterprise.views.quarantine_session')
     @mock.patch('enterprise.views.lift_quarantine')
     @mock.patch('enterprise.views.CourseApiClient')
-    def test_get_course_enrollment_page(
+    @mock.patch('enterprise.views.organizations_helpers')
+    @mock.patch('enterprise.views.CourseCatalogApiClient')
+    def test_get_course_specific_consent(
```

Comment:
@itsjeyd Unintended. This is me not being good at rebasing with git (or git at all...). Thanks for catching it.